### PR TITLE
🧿 Ajna empty page + navigation concept

### PR DESCRIPTION
**Pull request by piotrkonowrocki**
# 🧿 Ajna empty page + navigation concept

This is still working in progress, missing couple animations and probably might be refined or shortened in some places.

image
  
## Changes 👷‍♀️

- Created Ajna empty page hidden behind feature flag, with overwritten theme and new background,
- created set of new `Navigation` components that are abstract and can (and in the future should) be used to replace old navigation in Oasis.app,
- created `AjnaNavigationController` which imlements `Navigation` component for Ajna pages.
- created `AssetPill` component that is supposed to replace existing token pills on Oasis.app landing page (see: https://www.figma.com/file/XAUn9bc0mlzIS63d9yDUUF?node-id=1268:72387#326364235)
  
## How to test 🧪

Go to `/ajna` page with `Ajna` feature flag enabled. Without it there should be redirect to homepage.

### CODE REVIEW

File: components/navigation/NavigationMenuDropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+                    }}
+                  >
+                    Other assets you can borrow against
+                  </Heading>
+                  <Flex
+                    as="ul"
+                    sx={{ flexWrap: 'wrap', columnGap: 3, rowGap: 2, listStyle: 'none', p: 0 }}
+                  >
+                    {otherAssets.map(({ link, token }) => (
+                      <Box as="li">
```
Comment (piekczyk) [inline components/navigation/NavigationMenuDropdown.tsx:179]: key prop

File: components/navigation/NavigationMenuDropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+                as="ul"
+                sx={{
+                  gap: 4,
+                  gridTemplateColumns: 'repeat(2, 1fr)',
+                  p: '0',
+                  listStyle: 'none',
+                }}
+              >
+                {links.map(({ icon, footnote, link, title }) => (
+                  <Flex as="li">
```
Comment (piekczyk) [inline components/navigation/NavigationMenuDropdown.tsx:140]: key prop

File: components/navigation/NavigationMenuDropdown.tsx [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+            boxShadow: 'buttonMenu',
+            height: '600px',
+            ...(width > 0 && { width: width + 64 }),
+            ...(height > 0 && { height: height + 64 }),
+            p: 4,
+            ...(isPanelSwitched && { transition: 'width 200ms, height 200ms' }),
+          }}
+        >
+          {panels.map(({ description, label, learn, links, otherAssets }) => (
+            <Flex
```
Comment (piekczyk) [inline components/navigation/NavigationMenuDropdown.tsx:92]: key prop

**Review by piekczyk — CHANGES_REQUESTED** [review]

File: components/navigation/NavigationMenuDropdown.tsx (hunk 1 above)
Comment (piotrkonowrocki) [reply components/navigation/NavigationMenuDropdown.tsx]: Fixed...

File: components/navigation/NavigationMenuDropdown.tsx (hunk 2 above)
Comment (piotrkonowrocki) [reply components/navigation/NavigationMenuDropdown.tsx]: ...fixed...

File: components/navigation/NavigationMenuDropdown.tsx (hunk 3 above)
Comment (piotrkonowrocki) [reply components/navigation/NavigationMenuDropdown.tsx:92]: ...and fixed.

**Pull request merged**